critiquebrainz/frontend/external/musicbrainz.py

```diff
@@ -196,3 +217,14 @@ def get_entity_by_id(id, type='release_group'):
     elif type == 'place':
         rv = get_place_by_id(id)
     return rv
+
+
+def get_url_rels_from_releases(releases):
+    """Returns all url-rels for a list of releases in a single list (of url-rel dictionaries)
+    Typical usage with browse_releases()
+    """
+    all_url_rels = []
+    for release in releases['release-list']:
```

Comment:
I don't understand what the problem is. `releases['release-list']` is a list and you didn't expect it to be? Or is it something else?